client/my-sites/earn/memberships/add-edit-plan-modal.jsx

```diff
@@ -299,7 +300,7 @@ const RecurringPaymentsPlanAddEditModal = ( {
 					<h6 className="memberships__dialog-form-header">{ translate( 'Posts via email' ) }</h6>
 					<p>
 						{ translate(
-							'Allow members of this recurring payment plan to opt into receiving new posts via email.'
+							'Allow members of this payment plan to opt into receiving new posts via email.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.